client/my-sites/plugins/access-control.js

```diff
@@ -90,18 +102,24 @@ const hasRestrictedAccess = ( site ) => {
 		);
 	}
 
-	if ( abtest( 'businessPluginsNudge' ) === 'drake' && hasErrorCondition( site, 'noBusinessPlan' ) ) {
-		pluginPageError = {
-			title: i18n.translate( 'Want to add a store to your site?' ),
-			line: i18n.translate( 'Support for Shopify, Ecwid, and Gumroad is now available for WordPress.com Business.' ),
-			action: i18n.translate( 'Upgrade Now' ),
-			actionURL: '/plans/' + site.slug,
-			illustration: '/calypso/images/drake/drake-whoops.svg',
-			actionCallback: () => {
-				analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', { cta_name: 'business_plugins' } );
-			},
-			featureExample: getMockBusinessPluginItems()
-		};
+	if ( hasErrorCondition( site, 'noBusinessPlan' ) ) {
+		switch ( 'nudge' ) {
```

Comment:
Adding a link to my comment directly on the commit (as it might be missed by someone else looking at this): https://github.com/Automattic/wp-calypso/commit/e0632d7f2fea07620b3f373d0d6cffdb0c214b3b#commitcomment-15660207